docs/sphinx/FAQ.rst

```diff
@@ -145,6 +145,42 @@ Furthermore combined with holoviews, datashader can also be employed in an inter
 .. image:: _static/datashader.png
 
 
+.. raw:: html
+
+   </div>
+   </details>
+   <br>
+   <details>
+   <summary>
+      <a><b>Pandas or numpy datetime works much slower than unix epoch timestamps?</b></a>
+   </summary>
+   <div style="margin-left:1em">
+
+This stems from the plotly scatter(gl) constructor being much slower for non-numeric data. Plotly performs a different serialization for datetime arrays (which are interpreted as object arrays).
+However, plotly-resampler should not be limited by this - to avoid this issue, add your datetime data as `hf_x` to your plotly-resampler ``FigureResampler.add_trace`` (or ``FigureWidgetResampler.add_trace``) method.
+This avoids adding (& serializing) *all* the data to the scatter object. Now plotly-resampler will only add the necessary resampled data to the scatter object.
```

Comment:
```suggestion
This stems from the plotly scatter(gl) constructor being much slower for non-numeric data. Plotly performs a different serialization for datetime arrays (which are interpreted as object arrays).
However, plotly-resampler should not be limited by this - to avoid this issue, add your datetime data as `hf_x` to your plotly-resampler ``FigureResampler.add_trace`` (or ``FigureWidgetResampler.add_trace``) method.
This avoids adding (& serializing) *all* the data to the scatter object, since  plotly-resampler will pass the aggregated data to the scatter object.
```